### PR TITLE
Format code using 'cargo fmt' & add formatting check on CI

### PR DESCRIPTION
**Pull request by Atul9**

### CODE REVIEW

**Comment by piotrmaks** [comment]
I think it would make sense to enforce code format in CI.

**Comment by nox** [comment]
Thanks for the contribution but as Piotr says it has no purpose if it is not automated.

**Comment by Atul9** [comment]
@nox how about if I include the code formatting check on the CI? Can you please reopen the PR so I can push the commit?

**Comment by nox** [comment]
Sure.

**Comment by Atul9** [comment]
@nox I have updated the PR, please check

File: rustfmt.toml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-match_block_trailing_comma = true
+match_block_trailing_comma = false
```
Comment (nox) [inline rustfmt.toml:1]: Why did you change this?

Comment (Atul9) [reply rustfmt.toml:1]: `match_block_trailing_comma` is unstable feature which is only available on nightly channel.
When cargo fmt is run on stable channel, the job fails - https://travis-ci.org/github/nox/serde_urlencoded/jobs/715205245#L237

I have configured to run rustfmt to be run only on rust stable since rustfmt is sometimes not available on latest nightly channel.

Comment (nox) [reply rustfmt.toml:1]: Thanks for the explanation.
